### PR TITLE
new copy for prolong thankyou page

### LINKED ISSUE #686: Merci.js, copy update
Deliverables (Arjuna):
- [x] Copy

(Not corrected)
https://docs.google.com/document/d/1v24gadCvSiy2SMLMddAtgPHE7kEMu2CrPWOp3rMogsU/edit

### PR DESCRIPTION
**Pull request by ovbm**
after (I changed the h1 and h2 to be more inline with the welcome font on the account page):
Screenshot 2021-11-24 at 09 50 07
Screenshot 2021-11-24 at 09 49 51

before:
Screenshot 2021-11-24 at 09 53 32
Screenshot 2021-11-24 at 09 53 20

closes #686

### CODE REVIEW

File: components/Pledge/Merci.js [hunk 1]
```diff
@@ -280,15 +278,18 @@ class Merci extends Component {
...
-                  'merci/lead'
-                ])
-              }}
-            />
+            <Lead>
+              {t.first([
+                `merci/lead/package/${query.package || 'UNKOWN'}`,
+                'merci/lead'
+              ])}
+            </Lead>
```
Comment (tpreusse) [inline components/Pledge/Merci.js:286]: We still seem to use html and therefore need `RawHtml` for many packages:
Screenshot 2021-11-24 at 13 46 18

File: components/Pledge/Merci.js [hunk 2]
```diff
@@ -20,8 +20,8 @@ import { EMAIL_CONTACT, ONBOARDING_PACKAGES } from '../../lib/constants'
 
 import {
   A,
-  Interaction,
-  RawHtml,
+  H1,
+  P,
```
Comment (tpreusse) [inline components/Pledge/Merci.js:24]: `H1` and `P` are [legacy editorial elements from the crowdfunding](https://styleguide.republik.ch/typography#legacy-editorial-content-styles). Why switch from `Interaction`? Generally we use `Interaction` for selling (e.g. störer, angebot seite etc.)

Comment (tpreusse) [reply components/Pledge/Merci.js:24]: IMO we should eventually change account page to be sans serif / interaction as well.

File: components/Pledge/Merci.js (hunk 1 above)
Comment (ovbm) [reply components/Pledge/Merci.js:286]: Ah yeah. I didn't check with the old translations. hate the use of html in the translation strings. makes my head hurt XD

**Review by tpreusse — APPROVED** [review]

**Pull request merged**